packages/fx-core/tests/common/deps-checker/utils/dotnet.ts

```diff
@@ -0,0 +1,139 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import * as fs from "fs-extra";
+import * as path from "path";
+import * as os from "os";
+import * as tmp from "tmp";
+
+import { ConfigFolderName } from "@microsoft/teamsfx-api";
+import { cpUtils } from "../../../../src/common/deps-checker/util/cpUtils";
+import { isWindows } from "../../../../src/common/deps-checker/util/system";
+import { logger } from "../adapters/testLogger";
+import { createTmpDir } from "./common";
+import {
+  DotnetChecker,
+  DotnetVersion,
+} from "../../../../src/common/deps-checker/internal/dotnetChecker";
+
+const find = require("find-process");
+
+tmp.setGracefulCleanup();
+
+export const dotnetConfigPath = path.join(os.homedir(), "." + ConfigFolderName, "dotnet.json");
```

Comment:
Is this Unit Test or Integration Test?
If Unit Test, do not use home folder since this will break existing toolkit.
If Integration Test, separate these tests from existing core unit tests, as well as github action.

Reply:
Ts files that match "*.test.ts " is UT. `dotnet.ts` is a Util used in `dotnet.it.ts`.